job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -53,6 +53,16 @@ parameters:
     pattern: '^[^.,:;*?!|()]{1,48}$'
 
 paths:
+  /ping:
+    get:
+      tags:
+        - Jobs
+      summary: Ping Service
+      description: Ping Service
+      operationId: ping
+      responses:
+        '200':
+          description: OK
```

Comment:
Is there any error codes like 404 that maybe expected here

Reply:
Added 500 error code